zulipterminal/config/keys.py

```diff
@@ -1,304 +1,304 @@
 from collections import OrderedDict
-from typing import List, Set
+from typing import List, Tuple
 
 from typing_extensions import TypedDict
 
 
 KeyBinding = TypedDict('KeyBinding', {
-    'keys': Set[str],
+    'keys': Tuple[str, ...],
     'help_text': str,
     'excluded_from_random_tips': bool,
     'key_category': str,
 }, total=False)
 
 KEY_BINDINGS = OrderedDict([
     ('HELP', {
-        'keys': {'?'},
+        'keys': ('?',),
```

Comment:
While `mypy` should check for this during tests for development/released versions, if a user customizes this then this becomes a potential issue if they miss a trailing comma - for that reason I think a list would be simpler?

I do agree that immutable would be preferable, but the data types just don't always lend themselves to it, unfortunately.